core/spec/models/spree/price_spec.rb

```diff
@@ -96,6 +96,33 @@
         expect(price.country).to eq(country)
       end
     end
+
+    describe '.currently_valid' do
+      it 'prioritizes first those associated to a country' do
+        price_1 = create(:price, country: create(:country))
+        price_2 = create(:price, country: nil) { |price| price.touch }
+
+        result = described_class.currently_valid
+
+        expect(
+          result.index(price_1) < result.index(price_2)
```

Comment:
I think 

```ruby
expect(described_class.currently_valid).to eq [price_1, price_2]
```

has the same effect also taking the order into account. Do you prefer this version because it's more explicit?

Reply:
Yeah, it was my first try. The problem is that the fixtures are not very well optimized and other `Price` records get being created, so the result would contain other items. I think it should be food for another PR...